src/ucp/core/ucp_ep.h

```diff
@@ -71,10 +73,12 @@ typedef struct ucp_ep_config_key {
     } lanes[UCP_MAX_LANES];
 
     ucp_lane_index_t       am_lane;      /* Lane for AM (can be NULL) */
-    ucp_lane_index_t       rndv_lane;    /* Lane for zcopy Rendezvous (can be NULL) */
     ucp_lane_index_t       tag_lane;     /* Lane for tag matching offload (can be NULL) */
     ucp_lane_index_t       wireup_lane;  /* Lane for wireup messages (can be NULL) */
 
+    /* Lane for zcopy Rendezvous (can be NULL), sorted by priority, highest first */
```

Comment:
is there really a priority amongst the rndv lanes, or all are used?

Reply:
by default first N fastest lanes (for cases when user set limit of lanes below than available lanes)